dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/ValidationUtils.java

```diff
@@ -520,21 +519,19 @@ public static String valueIsValid( String value, DataElement dataElement, boolea
 
         OptionSet options = dataElement.getOptionSet();
 
-        boolean isMultiText = valueType == MULTI_TEXT;
-
-        if ( isMultiText && options == null )
+        if ( valueType.isMultiText() && options == null )
```

Comment:
This code looks a lot like the code in dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/validator/ValidationUtils.java not sure if we can/should make it reusable 🤷🏻

Reply:
We have a lot where aggregate and tracker are not collaborating. We can take this in a TECH issue.